libminifi/test/unit/StringUtilsTests.cpp

```diff
@@ -91,5 +95,19 @@ TEST_CASE("TestStringUtils::testEnv5", "[test split classname]") {
 
   std::string expected = "hello world ";
 
-  REQUIRE(expected == StringUtils::replaceEnvironmentVariables(test_string));
+  REQUIRE(expected == replaceEnvironmentVariables(test_string));
+}
+
+TEST_CASE("TestStringUtils::testJoin", "[test string join]") {
+  std::set<std::string> strings = {"3", "2", "1"};
+  REQUIRE(join(",", strings) == "1,2,3");
+
+  std::wstring sep = L"é";
+  std::vector<std::wstring> wstrings = {L"1", L"2"};
+  REQUIRE(join(sep, wstrings) == L"1é2");
+
+  std::list<uint64_t> ulist = {1, 2};
+  REQUIRE(join(sep, ulist) == L"1é2");
```

Comment:
what purpose does join serve here? The semantics of a join seem clear to allow strings.